samples/drivers/led_ws2812/Kconfig

```diff
@@ -6,4 +6,14 @@
 config SPI
 	default y
 
+menu "Sample Configuration"
+
+config LED_UPDATE_DELAY
```

Comment:
Probably a nit: not sure there is any specific guideline or best practice but would it be better to namespace it, ex. APP_LED_UPDATE_DELAY or SAMPLE_WS2812_LED_DELAY. Many (but not all) other samples seem to follow this approach.

Reply:
Good point. I did a `grep` and `SAMPLE` prefix seems a bit more common in occurrences and files count (and I personally also prefer this)